src/yamls/app.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: appstudio.redhat.com/v1alpha1
```

Comment:
Where would these yamls be used?

Reply:
Its just a sample to understand the APIs right now. We can remove them later.